docs/how-to-guides/meson-args.rst

```diff
@@ -85,33 +62,37 @@ To set this option temporarily at build-time:
 
         .. code-block:: console
 
-			 $ python -m build -Csetup-args="--default-library=static" .
+           $ python -m build -Csetup-args="--default-library=static" .
 
     .. tab-item:: pip
         :sync: key_pip
 
         .. code-block:: console
 
-			 $ python -m pip --config-settings=setup-args="--default-library=static" .
+	   $ python -m pip wheel . --config-settings=setup-args="--default-library=static" .
 
 
-2) Use Meson install_tags for selective installs
+Select the build targets to include in the wheel
 ------------------------------------------------
 
-Meson install_tags can be used (since ``meson-python`` >= 0.13) to select which
-targets are installed into the binary wheels. This example causes meson-python
-to only install targets tagged with ``runtime`` or ``python-runtime``) into the
-binary wheel (ignoring e.g. C++ headers).
+It is possible to include in the Python wheel only a subset of the
+installable files using Meson `installation tags`_ via the ``meson
```

Comment:
```suggestion
installable files using Meson `installation tags`_, via the ``meson
```

Reply:
No, a comma there is grammatically incorrect.